CHANGELOG.md

```diff
@@ -159,6 +159,11 @@ Indexes iterators names has been shortened to `Iter`, `Keys` and `Values`. (#162
 
 - `impl_serde_hex_for_binary_value` macro was moved from core to `merkledb`. (#1629)
 
+#### exonum-rust-runtime
+
+- Rust runtime module was moved from the `exonum` crate into the separate
```

Comment:
I think it should be as well noted in `exonum` section as breaking change and in `exonum-cli` as new feature.